api/justfile

```diff
@@ -50,6 +50,7 @@ up *flags:
 
 # Bring up services specific to the API profile, in addition to the API server
 up-extra *flags:
+    just up {{ flags }}
```

Comment:
If this is also calling `just ../up` with the correct compose profile, why should this also be called? If we needed it, shouldn't we just add `api` to the list of `COMPOSE_PROFILES` to spin up?

Reply:
You are right! I had not realised that. Fixed in 8379b8b.

Reply:
You are right! I had not realised that. Fixed in 8379b8b.